proposals/wac-ucr/index.bs

```diff
@@ -76,7 +80,23 @@ individuals (i.e. Alice, Bob, Carol, etc.) are already
 [=authenticated agents=], and their corresponding [=identities=] are known
 by the [=resource controller=] when they are granted access.
 
-## Basic resource access ## {#uc-basic}
+Additionally, since the vast majority of these use cases involve a
+[=resource controller=] reviewing and modifying permissions as an
+[=authenticated agent=], the following requirements are not listed explicitly
+in every use case, but should be assumed when permissions are assigned. 
+They may appear in certain use cases (such as [[#basic-change-permissions]]) when
+relevant.
+
+<div class="assertion">
+
+* [[#req-read-permissions]]
```

Comment:
This is "The system shall limit the ability to read the access permissions associated with a certain resource."
The problem may be with name of this requirement. The system "can limit access permissions associated with a certain resources". Or if one were to use the slightly heavy language "The system shall allow access permissions associated with a certain resource to be limited".

Reply:
Adjusted in https://github.com/solid/authorization-panel/pull/152/commits/691e677fb30249fd00b4d3a547899a98a7713f51